maxim/test_runs/run_utils.py

```
@@ -63,11 +73,13 @@ async def run_local_evaluations(
                 for name, result in combined_results.items():
-                    results.append(LocalEvaluationResult(
-                        name=name,
-                        pass_fail_criteria=evaluator.pass_fail_criteria[name],
-                        result=result
-                    ))
+                    results.append(
+                        LocalEvaluationResult(
+                            name=name,
+                            pass_fail_criteria=evaluator.pass_fail_criteria[name],
+                            result=result,
+                        )
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)